wallet/src/main/java/com/android/identity_credential/wallet/ProvisioningViewModel.kt

```diff
@@ -118,14 +127,14 @@ class ProvisioningViewModel : ViewModel() {
                 if (evidenceRequests!!.size == 0) {
                     state.value = State.PROOFING_COMPLETE
                     credential!!.let {
-                        it.refreshState(application.issuingAuthorityRepository)
+                        it.refreshState(issuingAuthorityRepository)
```

Comment:
Wanted to ask - is there a reason not to just do credential!!.refreshState(..) here?

Reply:
great catch! - my changes removed the `application` dependency but I could have also refactored the logic around it -- doing that now, thanks!